test/relationships.ts

```diff
@@ -1830,7 +1830,12 @@ test('self reference relation via column', async () => {
 })
 
 test('aggregate on missing column with alias', async () => {
-  const res = await selectQueries.aggregateOnMissingColumnWithAlias.eq('id', 1).limit(1).single()
+  const res = await selectQueries.aggregateOnMissingColumnWithAlias
+    // @ts-expect-error should not be able to eq 'id' since the column does not
+    // exist
+    .eq('id', 2)
```

Comment:
I think this test should never had passed because the query actually is made to return an error. And `id` column doesn't exist in `users` table.

We can make it pass tough by changing the `never` into an `unknown` here:

https://github.com/supabase/postgrest-js/pull/597/files#diff-8c09e973443260962548f07f6714d574185bce9a909a6d818691993e5a4253e5L43